### PR TITLE
11.0 mig partner contact nationality

### PR DESCRIPTION
**Pull request by jprobineau**

### CODE REVIEW

**Comment by njeudy** [comment]
@jprobineau did you work on this ? I can help or migrate

File: partner_contact_nationality/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
+# © 2017-Apertoso N.V. (<http://www.apertoso.be>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Contact nationality",
+    "summary": "Add nationality field to contacts",
+    'version': '11.0.1.0.0',
+    "category": "Customer Relationship Management",
+    "website": "https://odoo-community.org/",
```
Comment (nikul-serpentcs) [inline partner_contact_nationality/__manifest__.py:10]: @jprobineau IMHO Please change URL as described in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10

**Comment by NL66278** [comment]
Closing and reopening fixed the travis error (because partner_contact_personal_information_page  was merged some time ago). But runbot error for some reason remains...

**Comment by NL66278** [comment]
Still no succes with opening and closing.

File: partner_contact_nationality/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
```
Comment (NL66278) [inline partner_contact_nationality/__init__.py:1]: Remove copyright lines from init files and make license lines https:

File: partner_contact_nationality/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
+# © 2017-Apertoso N.V. (<http://www.apertoso.be>)
```
Comment (NL66278) [inline partner_contact_nationality/__manifest__.py:2]: Replace copyright sign with just the word `Copyright`. No need for (C) either.

**Review by NL66278 — CHANGES_REQUESTED** [review]
Usefull module, but needs a number of changes. See other comments.

**Comment by NL66278** [comment]
No action on comments and failing runbot

**Pull request closed without merging**